bagit.xml

```diff
@@ -389,12 +390,12 @@ ending           = CR / LF / CRLF
             The files under the payload directory are called payload files, or the payload.
             Each payload file is treated as an opaque octet stream when verifying file
             correctness.
-            Any sub-directory structure within the payload &must; be preserved but is
+            Any sub-directory structure within the payload &must-not; be changed but is
             otherwise ignored for purposes relating to this specification.
           </t>
         </section>
         <!-- /Payload Directory -->
-        <section title="Payload Manifest: manifest-&lt;algorithm&gt;.txt" anchor="sec-payload-manifest">
+        <section title="Payload Manifest: manifest-algorithm.txt" anchor="sec-payload-manifest">
```

Comment:
I'm mixed on this change since `manifest-` is the literal value but the algorithm name varies. When we were using all caps to denote field values I would have suggested `manifest-ALGORITHM.txt` but I'm not sure whether there's a better option now than hoping people read the text & examples. This probably isn't a big deal.

Reply:
Yeah, I had hoped to italicize, but seems that italics weren't allowed in title. Sigh.